### ISSUE TITLE
[508] Fix Claims Status Tool accessibility issues

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#37923]
## Description

@AngelaFowler82 met with me about some a11y problems in the CST:
- Main list of claims or appeals should have `h3` headers (currently `h2`s)
- When viewing details, the timeline may include disabled entries. These are still clickable and focusable, and shouldn't be
- The Mobile alert on the main list page is no longer closable

## Tasks

- [ ] Change header levels of main list
- [ ] Make mobile alert closable
- [ ] Disable button in timeline with no info
- [ ] Add unit tests

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing

**Comment by AngelaFowler82** [comment]
I can confirm that the issues listed in this ticket are fixed. I would add however that the headings on the individual claims pages are not nested properly. there should be an h3 between "What you've claimed" and "We're adding your details. and there should be an h3 "Claim status" above the steps in the process, and the steps should be h4s.

**Comment by Mottie** [comment]
LOL ok Angela.. thanks for catching that.
